web/ui/react-app/src/Panel.tsx

```diff
@@ -100,6 +101,7 @@ class Panel extends Component<PanelProps, PanelState> {
   }
 
   executeQuery = (expr: string): void => {
+    this.props.onExecuteQuery(expr)
```

Comment:
Nit: no semicolon (can we automate this via linter checks?)

Reply:
Yes, I can make such a setup.

Reply:
See https://github.com/prometheus/prometheus/issues/6199